torch/csrc/jit/script/module.h

```diff
@@ -59,6 +59,16 @@ struct Method {
     }
     get_executor().run(stack);
   }
+
+  IValue operator()(std::vector<IValue> inputs) {
+    run(inputs);
+    AT_CHECK(
```

Comment:
When the thing returns more than 1 value (or no values), let's auto-create a tuple:

```
if(inputs.size() != 1) {
   return IValue(Tuple::create(std::move(inputs));
}
```

This will match what we want in the future.